calibration/CTDMOG/CGINS-CTDMOG-13578__20170404.csv

```diff
@@ -14,12 +14,12 @@ serial,name,value,notes
 37-13578,CC_pa1,4.603066e-003,
 37-13578,CC_pa2,-3.748315e-011,
 37-13578,CC_ptca0,5.249251e+005,
-37-13578,CC_ptca1,8.283893e+000,
+37-13578,CC_ptca1,8.283894e+00,
 37-13578,CC_ptca2,3.086300e-001,
 37-13578,CC_ptcb0,2.522325e+001,
 37-13578,CC_ptcb1,2.500000e-004,
 37-13578,CC_ptcb2,0.00E+00,
 37-13578,CC_ptempa0,1.788275e+002,
 37-13578,CC_ptempa1,-7.023026e-002,
 37-13578,CC_ptempa2,-2.752989e-007,
-37-13578,CC_wbotc,8.6108e-007,
+37-13578,CC_wbotc,8.610782e-07,
```

Comment:
Looks good.

Reply:
Looks good (CD and JL) verified with QCT